Dockerfile.windows

```diff
@@ -262,15 +262,12 @@ RUN `
 RUN `
   Function Build-GoTestSum() { `
     Write-Host "INFO: Building gotestsum version $Env:GOTESTSUM_COMMIT in $Env:GOPATH"; `
-    $env:GO111MODULE = 'on'; `
-    &go get -d "gotest.tools/gotestsum@${Env:GOTESTSUM_COMMIT}"; `
-    $env:GO111MODULE = 'off'; `
-    if ($LASTEXITCODE -ne 0) {  `
-      Throw '"Failed getting gotestsum sources..."'  `
-    }; `
-    $env:GO111MODULE = 'on'; `
-    &go build -buildmode=exe -o "${Env:GOPATH}\bin\gotestsum.exe" gotest.tools/gotestsum; `
-    $env:GO111MODULE = 'off'; `
+    $Env:GO111MODULE = 'on'; `
+    $tmpGobin = "${Env:GOBIN_TMP}"; `
+    $Env:GOBIN = """${Env:GOPATH}`\bin"""; `
+    &go get -buildmode=exe "gotest.tools/gotestsum@${Env:GOTESTSUM_COMMIT}"; `
```

Comment:
Doesn't this one need `-ldflags` to set `main.version` too?

Reply:
Ah, yes, perhaps we should add it there (we didn't do that before, it was mostly when I tried checking if it's the right version)

Reply:
or perhaps I should remove it from the other one, as we don't really use it 🤔

let me know if you feel strongly about one or the other

Reply:
I don't really feel strongly either way -- just that we should try to keep them consistent. :sweat_smile:

I'm personally fine with this binary not knowing what version it is, but I don't run it very often, so I think I'm the wrong one to opine. :shrug: :heart:

Reply:
Agree, let's just be consistent.